main.tf

```diff
@@ -19,8 +31,8 @@ module "deploy" {
 
 provider "cml2" {
   address        = "https://${module.deploy.public_ip}"
-  username       = local.cfg.app.user
-  password       = local.cfg.app.pass
+  username       = local.cfg.secrets.app.username
+  password       = local.cfg.secrets.app.secret
```

Comment:
When things get moved to `local.cfg.secrets`, the `raw_secrets` get copied over to `.secrets` when using the `dummy` secrets manager.  I can't use `raw_secrets` for everything, because then Vault and Conjur won't work.  I hope this makes sense.